steps/install-bundler/1.2.0/step.yml

```diff
@@ -0,0 +1,39 @@
+title: Install specific bundler version
+summary: This step allows you to uninstall the system bundler version, and match the
+  one in your Gemfile.
+description: This step allows you to uninstall the system bundler version, and match
+  the one in your Gemfile..
+website: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+published_at: 2022-05-12T11:56:47.099737+02:00
+source:
+  git: https://github.com/FutureWorkshops/bitrise-step-install-bundler.git
+  commit: 2531a1fda45e317d478c6dd3c25c3efd14359010
+host_os_tags:
+- osx-10.14
+project_type_tags:
```

Comment:
Can't this step be useful on any platform? If so please remove the `project_type_tags` so that it will appear in the Step list of the workflow editor for all project types.

If you want to keep the restriction, i would like to ask you to remove `xamarin`, as [Xamarin support ended on Bitrise](https://blog.bitrise.io/post/xamarin-support-ends-in-2022-on-bitrise).